dotcom-rendering/src/components/Discussion/Comments.stories.tsx

```diff
@@ -129,109 +142,141 @@ InitialPage.decorators = [
 	]),
 ];
 
-export const LoggedInHiddenNoPicks = () => (
-	<div
-		css={css`
-			width: 100%;
-			max-width: 620px;
-		`}
-	>
-		<Comments
-			shortUrl="p/abc123"
-			isClosedForComments={false}
-			user={aUser}
-			baseUrl="https://discussion.theguardian.com/discussion-api"
-			additionalHeaders={{
-				'D2-X-UID': 'testD2Header',
-				'GU-Client': 'testClientHeader',
-			}}
-			expanded={false}
-			onPermalinkClick={() => {}}
-			apiKey=""
-			idApiUrl="https://idapi.theguardian.com"
-			page={3}
-			setPage={() => {}}
-			filters={filters}
-			commentCount={discussionMock.discussion.commentCount}
-			loading={false}
-			totalPages={discussionMock.pages}
-			comments={discussionMock.discussion.comments}
-			setComment={() => {}}
-			handleFilterChange={() => {}}
-		/>
-	</div>
-);
+export const LoggedInHiddenNoPicks = () => {
+	const [isActive, setActive] = useState(false);
```

Comment:
Now that we're moving to the reducer, is it still okay if I use `useState` in stories to stub these methods where needed, so that stories don't loose existing interactivity?

Reply:
I think that’s fine for now